packages/fx-core/src/core/index.ts

```diff
@@ -1552,7 +1551,7 @@ export function undefinedName(objs: any[], names: string[]) {
 }
 
 export function getProjectSettingsVersion() {
-  if (isFeatureFlagEnabled(FeatureFlagName.InsiderPreview, false)) return "2.0.0";
+  if (isMultiEnvEnabled()) return "2.0.0";
```

Comment:
So for the latest toolkit (v3.0.0), the project version in `projectSettings.json` is `2.0.0` rather than `3.0.0`?

Reply:
I know the issue. This PR mainly focuses on telemetry. The semantics of `version` will be in the next PR

Reply:
Cool! Thanks for the explaination.